core/inputs/input.ts

```diff
@@ -324,9 +324,17 @@ export class Input {
     if (!fieldRowLabel) {
       const inputs = this.getSourceBlock().inputList;
       const index = inputs.indexOf(this);
-      if (index > 0) {
```

Comment:
This seems necessary to keep (at least, `slice` will probably misbehave without this check). That being said, I'm not sure it's even possible for this scenario to logically happen in practice, but it's definitely possible just from an API perspective.